Common/UI/SubworldLoadingScreen.cs

```diff
@@ -24,6 +24,8 @@ internal static void DrawLoading(MappingWorld mappingWorld)
 		string statusText = Main.statusText;
 		GenerationProgress progress = WorldGenerator.CurrentGenerationProgress;
 
+		// The Player draw call doesn't layer properly if we don't specifcally replace the SpriteBatch like this -
+		// it's not anything functional otherwise.
```

Comment:
```suggestion
		// The Player draw call doesn't layer properly if we don't specifically replace the SpriteBatch like this -
		// it's not anything functional otherwise.
```